Cargo.lock

```diff
@@ -4141,6 +4219,17 @@ dependencies = [
  "lazy_static",
 ]
 
+[[package]]
+name = "time"
+version = "0.1.44"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "6db9e6914ab8b1ae1c260a4ae7a49b6c5611b40328a735b21862567685e73255"
+dependencies = [
+ "libc",
+ "wasi 0.10.0+wasi-snapshot-preview1",
+ "winapi",
+]
```

Comment:
Ahhhh noo! Back from the dead! Not sure what's bringing this in, but let's find it and squash it.

Reply:
it was tracing-subscriber.